cmd/prometheus/main.go

```diff
@@ -220,6 +226,12 @@ func main() {
 		os.Exit(2)
 	}
 
+	cfg.web.CORSOrigin, err = compileCORSRegexString(cfg.corsRegexString)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, errors.Wrapf(err, "could not compile regex string %q", cfg.corsRegexString))
```

Comment:
I think you meant "... should probably **not** mention ..." , I made the changes accordingly.
Should I update the docs here as well? 

https://prometheus.io/docs/operating/security/

> For non-mutating endpoints, you may wish to set CORS headers such as Access-Control-Allow-Origin in your reverse proxy to prevent XSS or use the `web.cors.origin` option.

Reply:
I meant what I said.

> Should I update the docs here as well?

You can once this gets released, it's over in the docs repo.